minorminer/layout/tests/__init__.py

```diff
@@ -0,0 +1,51 @@
+import unittest
```

Comment:
not a huge fan of this being named `__init__.py` -- perhaps name it `common.py` or something?

Reply:
I can move it to a file `common.py` but `unittest` can't discover tests unless there is an `__init__.py` (but a blank one is fine).

Reply:
yeah, a blank one is good